### PR TITLE
feat(addon-doc): add copy button out-of-the-box

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the new behavior?

image

### CODE REVIEW

**Review by waterplea — APPROVED** [review]
Let's also make it visible when `:focus-visible` so keyboard navigation looks good.

File: projects/addon-doc/components/example/example.style.less [hunk 1]
```diff
@@ -78,10 +78,6 @@
     display: flex;
     justify-content: flex-end;
     align-items: center;
-
-    & > * {
-        margin-left: 0.5rem;
```
Comment (waterplea) [inline projects/addon-doc/components/example/example.style.less:83]: What was it for?

File: projects/addon-doc/components/code/code.style.less [hunk 2]
```diff
@@ -10,13 +10,27 @@
...
+.t-code:hover .t-copy {
+    opacity: 1;
+}
+
+.t-copy {
+    position: absolute;
+    top: 0.875rem;
+    right: 2rem;
+    opacity: 0;
+    transition: opacity 200ms ease-in;
```
Comment (waterplea) [inline projects/addon-doc/components/code/code.style.less:31]: Why custom transition?

Comment (splincode) [reply projects/addon-doc/components/code/code.style.less]: fixed

File: projects/addon-doc/components/example/example.style.less (hunk 1 above)
Comment (splincode) [reply projects/addon-doc/components/example/example.style.less]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Comment by splincode** [comment]
@vladimirpotekhin fixed

**Pull request merged**